pyproject.toml

```diff
@@ -34,18 +34,20 @@ dependencies = [
 [project.optional-dependencies]
 test = [
     "codecov",
+    "fastparquet",
```

Comment:
Sorry, mixed two changes here (sorting, and adding some additional dependencies like pandas-stubs and mypy)